100/pick.ml

```diff
@@ -3650,7 +3650,7 @@ let PICK = prove
   REWRITE_TAC[SET_RULE
    `{x | x IN (s UNION t) /\ P x} =
     {x | x IN s /\ P x} UNION {x | x IN t /\ P x}`] THEN
-  (CONV_TAC o GEN_SIMPLIFY_CONV TOP_DEPTH_SQCONV (basic_ss []) 5)
+  ((CONV_TAC "o") o GEN_SIMPLIFY_CONV TOP_DEPTH_SQCONV (basic_ss []) 5)
```

Comment:
This one isn't reasonably matched, though I'm not sure how it'd be covered by this technique.